executor/executor.yml

```diff
@@ -163,8 +170,8 @@
    executable: /bin/bash
   register: c_job_status
   until: c_job_status.stdout == "Complete"
-  delay: 1
-  retries: 300
+  delay: 10
+  retries: "{{ (chaos_monitoring_time.stdout)|int + 500}}"
```

Comment:
Just thought about this @amitbhatt818 . If our delay is >1 (of the tune of, say, 5-10ms), then we wouldn't need to add the additional default fixed seconds (500). As the {{ chaos_monitoring_time.stdout }} * {{ delay }} itself would be >> expected time for job completion. To understand better, let us take this example: 

TOTAL_CHAOS_DURATION: 600s

- With earlier case executor would terminate if job is still in progress
- With the new approach, max wait would be (600 + 500)* 10 -- 11000s (worst case/stuck jobs will need 3+ hours to recover/end.) 

I suppose an optimal estimation would be:

- If we decide to provide much larger window to account for post chaos stuck-ups (say every cleanup post chaos has a high wait interval), then ( {{ (chaos_monitoring_time.stdout) | int + 300s }} * 2s